tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -65,6 +73,40 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		if (DO_GOFMT && !fileName.startsWith(".") /* criterion taken from gofmt*/ && fileName.endsWith(".go")) {
```

Comment:
It gets called on generated files, only. That's what `genFile` is about. It is filtering here, because it should not filter `.token` files that are generated as well.